app/modules/sightings/models.py

```diff
@@ -697,7 +697,12 @@ def build_identification_request(
         return id_request
 
     def send_identification(
-        self, config_id, algorithm_id, annotation_uuid, annotation_sage_uuid
+        self,
+        config_id,
+        algorithm_id,
+        annotation_uuid,
+        annotation_sage_uuid,
+        matching_set_query_override=None,
```

Comment:
This would be simpler if it was changed to just pass the matching_set_query in all cases rather than the override.